website/docs/r/origin_pop.html.markdown

```diff
@@ -41,4 +42,4 @@ The following attributes are exported:
 
 ## Import
 
-Origin POP cannot be imported.
\ No newline at end of file
+terraform import incapsula_origin_pop.aws-east site_id/dc_id
```

Comment:
You should wrap this line with ```

For example: 

```
$ terraform import incapsula_origin_pop.aws-east site_id/dc_id
```